collections/_article/dev-snapshot-godot-2-1-5-rc-2.md

```diff
@@ -17,11 +17,11 @@ On top of that, I decided to wait for this RC2 to have a proper fix for the [And
 
 This is the second release candidate for 2.1.5, and if all goes well, it should be the last. For this I will need the help of all 2.1.x users to test this release, use it to export your existing projects and check that there are no regressions from 2.1.4.
 
-- [Downloads](https://downloads.tuxfamily.org/godotengine/2.1.5/rc2/)
-- [Changelog](https://downloads.tuxfamily.org/godotengine/2.1.5/rc2/Godot_v2.1.5-rc2_changelog.txt)
+- [Downloads](https://github.com/godotengine/godot-builds/releases/2.1.5-rc2/)
+- [Changelog](https://github.com/godotengine/godot-builds/releases/2.1.5/rc2-Godot_v2.1.5-rc2_changelog.txt)
```

Comment:
The changelog links seem to be broken.

Reply:
It seems like the regex replaced the wrong `/` with `-`. It should be `2.1.5-rc2/` and not `2.1.5/rc2-`.

Furthermore, we don't actually have 2.1.5-rc2 on godot-builds... we had nuked some old pre-release builds from TF to save space years ago, before we mirrored to GitHub, so some of those are lost to time.

We should probably do some dead link analysis on the blog content and clean up some of these old links.

Reply:
I've marked known dead links with strikethrough (`~~`) and have updated some links to be working, like the ones on the 2.1.6-stable blog post.